docker-compose.prod.yml

```diff
@@ -0,0 +1,28 @@
+version: '3.7'
+services:
+  bot:
+    image: godfather:alpha
+    restart: "always"
+    command: "start"
+    env_file: .env
+    ports:
+        - "4000:8080"
```

Comment:
Might want to put this port (4000) in the `.env`, just like `PGSQL_DATABASE_EXPOSED_PORT`